distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -63,6 +63,7 @@ export EXTRA_JAVA_OPTS
 # export KARAF_HOME # Karaf home folder
```

Comment:
❓ Did we confirm if karaf and karaf.bat have changed? We now package these scripts with our own modified version.

Reply:
there was a minor change to karaf.bat. https://github.com/apache/karaf/compare/karaf-4.1.3...karaf-4.1.4
I didn't see a karaf.bat in our repo, where would I find it.

Reply:
I don't think we version `bin/karaf` and `bin/karaf.bat` actually just the `bin/start` and `bin/start.bat` scripts. see: [ddf-common](https://github.com/codice/ddf/tree/2.11.x/distribution/ddf-common/src/main/resources/bin)

Reply:
@paouelle, @oconnormi, can you guys think of a way we could have some kind of test for this? It's really easy to miss and I'm sure it's going to bite us one day...

Reply:
In a previous project I worked on, we had to modify distributed files like this and we devise a build tool where we would check in the checksum of the original file and the diff for our change and at build time, we would first check that the original file still matches the same checksum and if so, we would apply the diff to it to get the modified file. If not then we would fail the build.